### PR TITLE
 #161966925 Implement article tagging functionalities

### PR DESCRIPTION
**Pull request by PromasterGuru**
#### What does this PR do?
- Allow users to tag their articles

#### Description of Task to be completed?
- Allow authenticated users to tag articles they author
- Allow users to view articles with the tags

#### How should this be manually tested?
```
$ https://github.com/andela/ah-infinity-stones.git
$ cd ah-infinity-stones
$ git checkout ft-tag-articles-161966925
$ virutalenv -m python3 venv
$ source venv/bin/activate
$ pip install -r requirements.txt
```
Test using postman withthe following endpoints
- POST /api/articles
- GET /api/articles/:slug

#### What are the relevant pivotal tracker stories?
[#161966925](https://www.pivotaltracker.com/story/show/161966925)

#### Screenshots (if appropriate)
screenshot 2018-12-19 at 11 08 49

screenshot 2018-12-19 at 11 08 23

### CODE REVIEW

File: authors/apps/articles/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
-# Generated by Django 2.1.3 on 2018-12-14 08:13
+# Generated by Django 2.1.3 on 2018-12-17 12:50
 
 from django.conf import settings
 from django.db import migrations, models
+import django.db.models.deletion
+import taggit.managers
```
Comment (amutava) [inline authors/apps/articles/migrations/0001_initial.py]: what does this import do/achieve?

Comment (PromasterGuru) [reply authors/apps/articles/migrations/0001_initial.py]: That is an autogenerated file created during migrations. The first importation is used when deleting models in the database while the second one helps in adding tags to the models.

File: authors/settings.py [hunk 2]
```diff
@@ -11,6 +11,9 @@
 """
 
 import os
+import dj_database_url
+
+import dj_database_url
```
Comment (r-wambui) [inline authors/settings.py]: Why do we have all this duplication

Comment (PromasterGuru) [reply authors/settings.py]: I have worked that. Thank you for pointing that out.

**Pull request merged**